examples/tutorial/x/blog/handlers.go

```diff
@@ -90,7 +90,7 @@ func (h CreateBlogMsgHandler) validate(ctx weave.Context, db weave.KVStore, tx w
 	// Retrieve tx main signer in this context
 	sender := x.MainSigner(ctx, h.auth)
 	if sender == nil {
-		return nil, ErrUnauthorisedBlogAuthor(nil)
+		return nil, errors.ErrUnauthorized.Newf(unauthorisedBlogAuthor, nil)
```

Comment:
This is odd to me, to see a Printf type function, but not the string, so I don't even know if the next arguments are correct. Can you inline these definitions that use Newf.

For those using New, it is your call to inline or keep them in vars.